config/user.role.anonymous.yml

```diff
@@ -14,6 +16,7 @@ weight: 0
 is_admin: false
 permissions:
   - 'access content'
+  - 'access content samples'
```

Comment:
No issues for CD, but this made me wonder how it would work with VRT for other sites - we wouldn't normally give this permission for not logged-in users, would we?

Reply:
True. In most or all other cases, we're limiting the Sample content to viewable by Content Edtitor role and managed by admin